tests/func/user.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
You already added it in `tests/unit/remote/ssh`, why copy paste it? Just put it somewhere where both tests can find it and use it.

Reply:
DONE